go-controller/pkg/config/config.go

```diff
@@ -593,6 +599,27 @@ var CommonFlags = []cli.Flag{
 		Destination: &cliConfig.Logging.ACLLoggingRateLimit,
 		Value:       20,
 	},
+	&cli.StringFlag{
+		Name:  "netflow-targets",
+		Value: Default.NetFlowTargets,
+		Usage: "A comma separated set of NetFlow collectors to export flow data (eg, \"10.128.0.150:2056,10.0.0.151:2056\")." +
+			"Each entry is given in the form [IP address:port]",
+		Destination: &cliConfig.Default.NetFlowTargets,
+	},
+	&cli.StringFlag{
+		Name:  "sflow-targets",
+		Value: Default.SFlowTargets,
+		Usage: "A comma separated set of SFlow collectors to export flow data (eg, \"10.128.0.150:6343,10.0.0.151:6343\")." +
+			"Each entry is given in the form [IP address:port]",
+		Destination: &cliConfig.Default.SFlowTargets,
+	},
+	&cli.StringFlag{
+		Name:  "ipfix-targets",
+		Value: Default.IPFIXTargets,
+		Usage: "A comma separated set of IPFIX collectors to export flow data (eg, \"10.128.0.150:2055,10.0.0.151:2055\")." +
+			"Each entry is given in the form [IP address:port]",
+		Destination: &cliConfig.Default.IPFIXTargets,
+	},
```

Comment:
I don't see any verification to arguments before passing them to ovsctl ?

Reply:
that's fair, we do make a verification upstream via CRD in OpenShift, I do not here, so I can add.
As for IPv4 or IPv6 docs just show IPv4.

Reply:
done

Reply:
not for this PR but could be future enhancement ? I see in the code we have ability to configure ovn via ConfigMap here are some of the fields we program with cli or configmap
```
# The network cidr and service cidr are set in the ovn-config configmap
kind: ConfigMap
apiVersion: v1
metadata:
  name: ovn-config
  namespace: ovn-kubernetes
data:
  net_cidr:      "10.244.0.0/16"
  svc_cidr:      "10.96.0.0/16"
  k8s_apiserver: "https://ovn-control-plane:6443"
  mtu:           "1400"
```

Reply:
Wrong layer. If you grep the code, you will see the ovn-config just contains K8S control plane related data, e.g. pod CIDR, service CIDR, etc.
Optional features that are not in the layer of 'required config for K8S' are put elsewhere, namely as envvars.
As for CRD, that's too overkill for this, we don't need a reconcile loop to keep track of state of a string containing a list of hostports.

Reply:
MTU is there and its not k8s , I think you will agree with me in passing the flows via command line isn't user friendly having some sort of yaml like networkpolicy probably will be more preferred, as I said nothing required in this PR but just some thoughts to help usability

Reply:
Right, is not k8s, but it's a fundamental value that is needed for the OVN to work.
WHat i'm saying is that if you see the rest of the code, none of the parameters is in the ovn-config ConfigMap, thus it makes no sense to put it there.
A networkpolicy is a way more complex object than is not easily defined as something as a string, thus it has its CRD.
Not everything is subject to become a CRD for the sake of it, just when it makes sense.

Reply:
> MTU is there and its not k8s , I think you will agree with me in passing the flows via command line isn't user friendly having some sort of yaml like networkpolicy probably will be more preferred, as I said nothing required in this PR but just some thoughts to help usability

Also, I think you are confusing things. I'm not passing the flows here. The parameter is list of the collector IPs and ports, then OVS send the flow metadata to those collectors by using NetFlow/SFlow/IPFIX